model_analyzer/triton/server/server_docker.py

```diff
@@ -68,8 +68,11 @@ def __init__(self, image, config, gpus, log_path, mounts, labels, shm_size):
         assert self._server_config['model-repository'], \
             "Triton Server requires --model-repository argument to be set."
 
-        logger.info(f"Pulling docker image {self._tritonserver_image}")
-        self._docker_client.images.pull(self._tritonserver_image)
+        try:
```

Comment:
I didn't see any way in the python docker module to say "does this image exist". I could have just put a try around the `pull`, but this seems a little more obvious what the code is doing, and will suppress the `pulling docker image` printout when the image exists locally (either because it was built locally, or it was already pulled)